### PR TITLE
intercept kubelet get node request in order to reduce the traffic

### PR DESCRIPTION
**Pull request by vie-serendipity**
/kind feature
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by vie-serendipity** [comment]
@rambohe-ch I add a check whether it's the first request after reconnection.

File: pkg/yurthub/cachemanager/cache_manager.go [hunk 1]
```diff
@@ -433,9 +428,13 @@ func (cm *cacheManager) saveWatchObject(ctx context.Context, info *apirequest.Re
 				} else {
 					updateObjCnt++
 				}
+				errMsg := cm.updateInMemoryCache(ctx, info, obj)
+				klog.ErrorS(errMsg, "failed to update cache")
```
Comment (rambohe-ch) [inline pkg/yurthub/cachemanager/cache_manager.go:432]: only print error message only when error happened.

**Pull request merged**